resources/i18n/en.json

```diff
@@ -307,6 +307,11 @@
                 "MEDIUM": "1",
                 "HIGH": "2"
             },
+            "TIMER_TYPES": {
```

Comment:
This resource file should not be used any more, unfortunately it was left here, but the resource file, belonging to the module should be updated.